app/test_main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import pytest
+from unittest.mock import patch
```

Comment:
check order of imports, also different import types must be separated with blank lines